api/src/wfl/module/all.clj

```diff
@@ -111,7 +112,7 @@
 (s/def ::pipeline string?)
 (s/def ::project string?)
 (s/def ::release string?)
-(s/def ::status (set (conj cromwell/statuses "skipped")))
+(s/def ::status (set (conj (distinct (into cromwell/statuses rawls/statuses)) "skipped")))
```

Comment:
Maybe: `(s/def ::status (set (concat cromwell/statuses rawls/statuses ["skipped"])))`
`set` will `distinct` for you.
(I wish we could get rid of `"skipped"`.)